### PR TITLE
Added swift codes to iOS quick start

### PR DESCRIPTION
**Pull request by tensiuyan**
I have added swift codes to the iOS quick start

### CODE REVIEW

**Comment by tensiuyan** [comment]
this PR fixes #466 

@ben181231 can you help check the codes and see if everything is okay?

**Comment by chpapa** [comment]
If @ben181231 approve the review request, I will merge it :)

File: content/quickstart/ios.md [hunk 1]
```diff
@@ -104,6 +111,31 @@ Practically the codes should not be structured this way. It is for demo only.
...
+// https://docs.skygear.io/guides/auth/basics/ios/
+
+let skygear = SKYContainer.default()
+skygear?.signupAnonymously(completionHandler: { (user, error) in
+    if error != nil {
+        print("Signup Error: \(error?.localizedDescription)")
+        return
+    }
+    
+    // Create the table "test" and the record "Hello world"
```
Comment (ben181231) [inline content/quickstart/ios.md]: I thing is this not good to mention "table" in the frontend code.

Comment (chpapa) [inline content/quickstart/ios.md]: I think we can address it more clearly. Using both Skygear's term (like Record type) and explain what's happening behind the scene.

**Pull request closed without merging**